### PR TITLE
.NET Core RC2 support

### PR DESCRIPTION
**Pull request by lstratman**

### CODE REVIEW

File: lib/edge-cs.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 var path = require('path');
 
 exports.getCompiler = function () {
-	return process.env.EDGE_CS_NATIVE || (process.env.EDGE_USE_CORECLR ? path.join(__dirname, 'edge-cs-coreclr', 'edge-cs-coreclr.dll') : path.join(__dirname, 'edge-cs.dll'));
+	return process.env.EDGE_CS_NATIVE || (process.env.EDGE_USE_CORECLR ? 'Edge.js.CSharp' : path.join(__dirname, 'edge-cs.dll'));
 };
+
+exports.getBootstrapDependencyManifest = function() {
+	return path.join(__dirname, 'bootstrap', 'bin', 'Release', 'netstandard1.5', 'bootstrap.deps.json');
+}
```
Comment (tjanczuk) [inline lib/edge-cs.js]: This is now an extended contract of an Edge compiler? Could you describe it at https://github.com/tjanczuk/edge/wiki/Add-support-for-a-CLR-language?

**Comment by tjanczuk** [comment]
@lstratman What is the full process for building and releasing edge-cs?

**Comment by tjanczuk** [comment]
@lstratman What is the dev workflow for building edge-cs after these changes?

**Comment by lstratman** [comment]
1. Build the .NET 4.0 assembly in src/edge-cs/ and copy it to lib/
2. Build the .NET Core NuGet package src/Edge.js.CSharp/ via: dotnet pack
--configuration Release
3. Upload the NuGet package to nuget.org
4. Publish the npm package

**Comment by tjanczuk** [comment]
https://www.nuget.org/packages/Edge.js.CSharp/1.0.0

**Comment by tjanczuk** [comment]
Published to NPM too: https://www.npmjs.com/package/edge-cs

**Pull request merged**